applications/SwimmingDEMApplication/custom_python/add_custom_strategies_to_python.cpp

```diff
@@ -63,9 +63,6 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #include "custom_strategies/strategies/adams_bashforth_strategy.h"
 #include "custom_strategies/strategies/residualbased_derivative_recovery_strategy.h"
 
-// convergence criteria
```

Comment:
If removing this line does not affect compilation I would say go ahead from the DEM Team.